database/migrations/2020_05_20_100000_create_contacts_table.php

```diff
@@ -12,7 +12,7 @@ public function up()
     {
         Schema::create('contacts', function (Blueprint $table) {
             $table->id();
-            $table->string('form_type')->index(); // String title of the form type so know which one was completed
+            $table->string('form_type')->index(); // Todo: Change this to enum?
```

Comment:
@drewroberts I believe we discussed making this an enum. Correct me if I'm wrong, but is this something that should be handled that way? After looking through how the controller handles this field, it looks seems like this is just treated as a path slug that's just used during redirecting to controllers. Would enum be the best approach if this is something that would be subject to frequent changes?

Reply:
Yes, I prefer enum for this. Look at how we use "Applies To" on discount:

- https://github.com/tipoff/discounts/blob/main/src/Nova/Discount.php

It is in tipoff/support but you can see how that would be a similar approach to what we want here. Those forms should not change much and we'll eventually refactor so an application can add their own set of enums if they have different forms.

- https://github.com/tipoff/support/blob/main/src/Enums/AppliesTo.php

Reply:
Better example:

- https://github.com/tipoff/taxes/blob/main/src/Enum/TaxCode.php

Reply:
Look at how the tipoff/taxes package uses TaxCode and let's do the same thing here for FormType

Reply:
https://github.com/tipoff/forms/pull/43/commits/946bb7c80a42bc4fadc0e954ffdbb04e6d920d49